rj_gameplay/rj_gameplay/skill/mark.py

```diff
@@ -11,9 +11,36 @@
 import stp.skill as skill
 import stp.role as role
 import stp.action as action
+from rj_gameplay.action import move
 from stp.skill.action_behavior import ActionBehavior
-from stp.skill.cost_behavior import CostBehavior
 import stp.rc as rc
+import stp.utils.constants as constants
+
+from rj_geometry_msgs.msg import Point, Segment
+
+def get_mark_point(target_robot_id: int, world_state: rc.WorldState):
```

Comment:
I'm assuming this is the wrong place to put this "cost function", but I don't know where the right place is.

Reply:
Okay, we can refactor later.